index.js

```diff
@@ -119,6 +119,120 @@ function closeModal(i) {
   document.querySelector(`.modal-${i}`).style.display = 'none';
 }
 
+// work section 1.
+
+const modalsOb = [
+  {
+    modalTitl: 'Components website',
+    modalLis: ['css', 'html', 'Bootstrap', 'Ruby'],
+    modalIm: './images/Img Placeholder.png',
+    modalTex:
+      "A daily selection of privately personalized reads; no accounts or sign-ups required. This has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text.",
+  }];
```

Comment:
- **[OPTIONAL]** Kindly correct the pop-up window position for a better user interface, align the modal window to the center and use `overflow-y: scroll;` if the height exceeded the screen size.

| Figma design | Your design |
|---|---|
| ![image](https://user-images.githubusercontent.com/58642949/216209506-9628b425-5374-45c4-9e64-c91cef8a37f6.png) | ![image](https://user-images.githubusercontent.com/58642949/216209566-4b305947-1f32-468f-a4cc-4e4a673c0d8b.png) |

- **[OPTIONAL]** Please 🙏🏽 avoid the horizontal scroll on the window.